src/main/java/com/rackspace/ceres/app/services/MetadataService.java

```diff
@@ -28,6 +28,8 @@
 import com.rackspace.ceres.app.utils.DateTimeUtils;
 import io.micrometer.core.instrument.Counter;
 import io.micrometer.core.instrument.MeterRegistry;
+import java.util.HashMap;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Do you really need to create this dependency on apache? What about standard classes?

Reply:
Yeahhh.... I was also kinda thinking that. @shabd67 you can use spring utils also here.

Reply:
sure